rkt/image_gc.go

```diff
@@ -83,7 +83,7 @@ func gcTreeStore(s *store.Store) error {
 	defer keyLock.Close()
 	referencedTreeStoreIDs, err := getReferencedTreeStoreIDs()
 	if err != nil {
-		return errwrap.Wrap(errors.New("cannot get referenced treestoreIDs"), err)
+		stderr.PrintE("cannot get referenced treestoreIDs", err)
```

Comment:
If `err != nil`, `referencedTreeStoreIDs` is nil, so the code below will call `RemoveTreeStore` on too many images.

Shouldn't you do that change in the function `getReferencedTreeStoreIDs()/walkPods()` instead?